src/Google.Storage.V1/StorageClient.UpdateObject.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Download;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = Google.Apis.Storage.v1.Data.Object;
+
+namespace Google.Storage.V1
+{
+    public abstract partial class StorageClient
+    {
+        /// <summary>
+        /// Updates the metadata for an object in storage synchronously.
+        /// </summary>
+        /// <remarks>
+        /// As this is a full update, <paramref name="obj"/> must be fully populated. This is typically
+        /// obtained by performing another operation (such as <see cref="GetObject(string, string, GetObjectOptions)"/>
+        /// with a "full" projection, and then modifying the returned object.
+        /// </remarks>
+        /// <param name="obj">Object to update. Must not be null, and must have populated <code>Name</code>,
+        /// <code>Bucket</code> and <code>Acl</code> properties.</param>
```

Comment:
Given our recent discussion, these should be <c>, not <code> (and many elsewhere)
Not sure if this actually matters.